### ISSUE TITLE
Sources left in article text after filtering

### ISSUE THREAD

**Issue by gjost** [denshoproject/encyc-core#31]
Article pages look like they have the proper images in the right sidebar but remnants of the images ("200px 200px 200px") are left in the main body text area, between header and start of text.
Example: https://encycstage.densho.org/Yuri%20Kochiyama

**Comment by gjost** [comment]
The empty links are present in the Elasticsearch record (http://192.168.1.100:9200/encycarticle/_doc/Yuri%20Kochiyama)
```
<div class=\"floatright\">\n <a class=\"new\" href=\"/index.php?title=Special:Upload&amp;wpDestFile=En-denshovh-kyuri-01-0004-1.jpg\" title=\"File:En-denshovh-kyuri-01-0004-1.jpg\">\n  200px\n </a>\n</div>\n<div class=\"floatright\">\n <a class=\"new\" href=\"/index.php?title=Special:Upload&amp;wpDestFile=En-denshovh-kyuri-01-0014-1.jpg\" title=\"File:En-denshovh-kyuri-01-0014-1.jpg\">\n  200px\n </a>\n</div>\n<div class=\"floatright\">\n <a class=\"new\" href=\"/index.php?title=Special:Upload&amp;wpDestFile=En-denshovh-kyuri-01-0011-1.jpg\" title=\"File:En-denshovh-kyuri-01-0011-1.jpg\">\n  200px\n </a>\n</div>\n
```

**Comment by gjost** [comment]
Fixed in `encyc-core` commit `77cec71` for package `encyccore-master_4.1.0~deb9`.
